core/src/helpers.rs

```diff
@@ -449,7 +455,11 @@ pub fn decode_execute_at_address_return(
 
 pub fn starknet_tx_into_eth_tx(
     tx: StarknetTransaction,
+    block_hash: FieldElement,
```

Comment:
let's make both `block_hash` and `block_number` Option types, that way they can be set to None instead of Zeroes when they're not available

Reply:
Done